### PR TITLE
ASA: Combustible land summary

### PR DESCRIPTION
**Pull request by conbrad**
# Test Links:
[Percentile Calculator](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2608.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/fba/components/ZoneSummaryPanel.tsx [hunk 1]
```diff
@@ -10,24 +12,28 @@ const useStyles = makeStyles({
...
-        </div>
-      </div>
-    )
+    return <></>
   }
+
+  return (
+    <Grid container alignItems={'center'} direction={'column'} spacing={2} className={classes.wrapper}>
+      <Grid item>
+        <Typography>Zone ID: {props.selectedZoneID}</Typography>
```
Comment (andrea-williams) [inline web/src/features/fba/components/ZoneSummaryPanel.tsx:28]: shouldn't need this line - the zone name will be displayed

Comment (conbrad) [reply web/src/features/fba/components/ZoneSummaryPanel.tsx:28]: Yeah I think the labels will change a lot, I'm gonna rebase off your PR once you get it in and adopt the labels that are there.

**Review by andrea-williams — APPROVED** [review]

**Review by dgboss — APPROVED** [review]
Is there already a ticket to adjust the color of the bars in the chart to match the orange/red of hfi areas?

**Comment by conbrad** [comment]
> Is there already a ticket to adjust the color of the bars in the chart to match the orange/red of hfi areas?

Nope but I can write one

**Pull request merged**